reinforcement_learning/rl_cartpole_ray/rl_cartpole_ray_gymEnv.ipynb

```diff
@@ -114,7 +114,11 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": [
+     "parameters"
```

Comment:
Does this mean `local_mode` also becomes a parameter? If yes, you may consider including it as part of the test.

Reply:
Good suggestion! Doing that as well.